packages/statusbar/src/tokens.ts

```diff
@@ -55,6 +55,11 @@ export namespace IStatusBar {
      */
     rank?: number;
 
+    /**
+     *  Displaying Items based on zoom priority -- higher zoom priority gets prioritised when zoom levels increase
+     */
+    priorityZoom?: number;
```

Comment:
Could we call it `priority`? The idea is in the future to use that information to tune the status bar items displayed with the window width to have a better reactive UI.

```suggestion
    priority?: number;
```